packages/antd/jest.config.js

```diff
@@ -6,7 +6,5 @@ module.exports = {
   },
   moduleNameMapper: {
     '\\.(css|less)$': '<rootDir>/__mocks__/styleMock.js',
-    '^react$': '<rootDir>/node_modules/react',
-    '^react-dom$': '<rootDir>/node_modules/react-dom',
```

Comment:
this is no longer correct or needed, instead there is single react version which is resolved correctly